src/ucp/api/ucp.h

```diff
@@ -1183,7 +1186,12 @@ typedef struct {
      * bits from @ref ucp_op_attr_t. Fields not specified in this mask will be
      * ignored. Provides ABI compatibility with respect to adding new fields.
      */
-    uint64_t       op_attr_mask;
+    uint32_t       op_attr_mask;
+
+    /**
+     * Operation specific flags
+     */
+    uint32_t       flags;
```

Comment:
What flags and where the flags are defined ?

Reply:
these flags will be define in the next pull requests (maybe we can move this field to other PR)
every operation defines its own flags:
- flush flags for ucp_ep_flush_nbx()
- active message flags for ucp_am_send_nbx()
- stream send flags for ucp_stream_send_nbx()

Reply:
Sounds like it does not belong to this PR.

Reply:
> Sounds like it does not belong to this PR.

Ok, we'll move to relevant PR

Reply:
@yosefe remove flags from this PR?

Reply:
yes

Reply:
done